modules/Kconfig

```diff
@@ -30,3 +34,23 @@ source "modules/Kconfig.tinycbor"
 source "modules/Kconfig.tinycrypt"
 source "modules/Kconfig.vega"
 source "modules/Kconfig.xtensa"
+
+comment "Unavailable modules, please install those via the project manifest."
+
+# List of Zephyr modules text when module is not available, please use the
+# following syntax:
+# ---------------------------------------------------
+# comment "<module_name> module not available."
+#	depends on !ZEPHYR_<MODULE_NAME_UPPER>_MODULE
+#
+# Remember to add the following code inside the `<module>/Kconfig.<module> file:
```

Comment:
`<module_root>/<module>/Kconfig.<module>` or have I misunderstood?

Reply:
This refers to this folder in which this file is used.

For external modules roots, that module is free to define how to handle this Kconfig stuff.
All modules natively supported in Zephyr uses this config to handle correct
```
comment "<module_name> module not available."
```
presentation, when module is not available.
But custom modules not natively in Zephyr is free not to do this.